framework/doc/content/source/bcs/ADPenaltyDirichletBC.md

```diff
@@ -0,0 +1,83 @@
+# ADPenaltyDirichletBC
+
+!syntax description /BCs/ADPenaltyDirichletBC
+
+## Description
+
+`ADPenaltyDirichletBC` is a `ADNodalBC` used for enforcing Dirichlet boundary conditions
```

Comment:
`Nodal` -> `Integrated`. You scared me for a second

Reply:
whoops, nice. This is just copied from the non-AD text. I'll update both.